workspaces/ui-v2/src/entry-points/cloud/cloud-viewer.tsx

```diff
@@ -163,36 +163,6 @@ export default function CloudViewer() {
   );
 }
 
-class CloudInMemorySpectacle
```

Comment:
Removed this since it's a full copy paste of `InMemorySpectacle` and to fix the type signature here would have involved adding graphql as a dependency for the ui-v2 instance (for reasons)